### PR TITLE
feat: removes lookup jsons from repo, using drei-assets instead

### PR DESCRIPTION
**Pull request by gsimone**
This removes the two jsons from the repo, fetching them from `drei-assets` instead.
I also moved the normal textures from @emmelleppi's repo to drei-assets.

I left the matcap repo alone because it's too much >stuff<

Note that this is not a breaking change as the use*Texture hooks were already suspense-based, so win win

### CODE REVIEW

File: src/core/useMatcapTexture.tsx [hunk 1]
```diff
@@ -20,11 +19,20 @@ function getFormatString(format: number) {
   }
 }
 
+const LIST_URL = 'https://cdn.jsdelivr.net/gh/pmndrs/drei-assets@master/matcaps.json'
 const MATCAP_ROOT = 'https://rawcdn.githack.com/emmelleppi/matcaps/9b36ccaaf0a24881a39062d05566c9e92be4aa0d'
```
[the remark this answers is not in the document]
Comment (gsimone) [reply src/core/useMatcapTexture.tsx:23]: I wouldn't want to re-release drei if we make changes to the assets, we are kind of using gh as a database 🤷

**Comment by gsimone** [comment]
on another note, I'd like to streamline how we use githack/jsdelivr etc, will accept suggestions

**Comment by CodyJasonBennett** [comment]
Is this too much to put on the market?

**Comment by gsimone** [comment]
We'd still have to make a lookup json and put it somewhere though, right?

**Review by joshuaellis — APPROVED** [review]

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**